flake8_isort.py

```diff
@@ -59,8 +70,29 @@ def run(self):
                     )
                 else:
                     sort_result = SortImports(self.filename, check=True)
-            if sort_result.incorrectly_sorted:
-                yield 0, 0, self.isort_error_msg, type(self)
+
+            for result in self.parse_line_num(sort_result):
+                yield result
+
+    def parse_line_num(self, sort_result):
+            self._cleanup_eof_blank_lines(sort_result)
+
+            differ = Differ()
+            diff = differ.compare(sort_result.in_lines, sort_result.out_lines)
+
+            line_num = 0
+            for line in diff:
+                if line.startswith('  ', 0, 2):
+                    line_num += 1  # Ignore unchanged lines but incr line no.
+
+                if line.startswith('- ', 0, 2):
```

Comment:
can this be an ``elif``? or otherwise on the previous ``if`` put a continue after the ``line_num += 1``